packages/adapter-node/tsconfig.json

```diff
@@ -13,5 +13,6 @@
 			"@sveltejs/kit": ["../kit/types/index"]
 		}
 	},
-	"include": ["index.js", "src/**/*.js", "internal.d.ts"]
+	"include": ["index.js", "src/**/*.js", "tests/**/*.js", "internal.d.ts"],
+	"exclude": ["tests/smoke.spec_disabled.js"]
```

Comment:
excluding `smoke.spec_disabled.js` from tsc since it's currently broken and fixing it is out of scope for this PR